routes.js

```diff
@@ -43,28 +28,15 @@ function fastifySwagger (fastify, opts, next) {
     url: '/documentation',
     method: 'GET',
     schema: { hide: true },
-    handler: (request, reply) => reply.redirect(request.raw.url + '/')
+    handler: (request, reply) => reply.redirect('./documentation/')
```

Comment:
Why the dot in `'./documentation/'`?

Reply:
so my case was the next, I serve my service under the reverse-proxy, like:
`/service` points to the service itself, let's say `localhost:3000`.

With the previous version, when I was visiting `/service/documentation` it was redirecting me to the `/documentation/`. With the current version it works fine.

Here how I've checked it:
```js
'use strict'

const fastify = require('fastify')()
const proxy = require('fastify-http-proxy')

fastify.register(require('../index'), {
  mode: 'static',
  specification: {
    path: './examples/example-static-specification.yaml'
  },
  exposeRoute: true
})

fastify.register(proxy, {
  upstream: 'http://localhost:3000/',
  prefix: '/upstream/' // optional
})

fastify.listen(3000, err => {
  if (err) throw err
  console.log('listening')
})
```
(put to the script into `examples`, install `fastify-http-proxy`, open `http://localhost:3000/upstream/documentation`)

Reply:
I think this is ok.